### PR TITLE
added button component

### PR DESCRIPTION
**Pull request by Manpreet-hub**
Added button component.
ShareX_Hf95O8J5Yq

### CODE REVIEW

File: Components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.btn-default{
```
Comment (anshikaa12) [inline Components/button/button.css:1]: you might want to consider giving a min width to button as I could see for small texts it turns out to be very small as it is taking the width of content inside .

File: docsite/docs.html [hunk 2]
```diff
@@ -160,8 +160,37 @@ <h3 class="txt-header-2">Badge on Icon</h3>
...
+    style="width: 730px; height: 298px; border:0; transform: scale(1); overflow:hidden;"
+    sandbox="allow-scripts allow-same-origin">
+  </iframe>></div>
+
+        <h3 class="txt-header-2">Floating action Button</h3>
+        <a class="btn-default btn-primary btn-float-action"><i class="fas fa-plus"></i></a>
```
Comment (anshikaa12) [inline docsite/docs.html:181]: add appropriate css to floating button it looks elliptical and not circular!

**Review by anshikaa12 — CHANGES_REQUESTED** [review]
look at the changes i suggested rest looks fine

File: Components/button/button.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    border: .1rem solid var(--voilet-color);
+}
+
+.btn-primary:hover{
+    color: var(--white-color);
+    background-color: var(--voilet-color);
+}
+
+.btn-float-action{
+    border-radius: 50%;
```
Comment (way2nizam) [inline Components/button/button.css:55]: here I have a suggestion for the float button, you should define two classes with left and right, and using "position: absolute" you can use that button.

**Review by way2nizam — COMMENTED** [review]
added some suggestion

File: docsite/docs.html (hunk 2 above)
Comment (Manpreet-hub) [reply docsite/docs.html:181]: updated CSS for floating button 
Screenshot (92)

Comment (Manpreet-hub) [reply docsite/docs.html:181]: updated css for floating button.

File: Components/button/button.css (hunk 3 above)
Comment (Manpreet-hub) [reply Components/button/button.css:55]: If I use position: absolute then floating buttons are laid at top of the page. I have updated CSS for floating btn. Please review it.

**Review by anshikaa12 — APPROVED** [review]

**Pull request merged**